### DISCUSSION TITLE
what were the tough decisions for you as creator when it comes sustainability you had to take

### DISCUSSION THREAD

**Discussion by MAJIDNISAR** [github/india#17] [Open Source] [answered]
sustainability ...

**Comment by liyasthomas** [comment, marked as answer]
Initial times are always the hardest. When starting out you've to find means on finding motivation, finance, and support to keep working on it. Always keep to minimise your expenses. Use free tool and tiers first and only scale to paid tiers when in need. Platforms like GitHub, Netlify etc offers an "Open source" tier to help developers who are in need. Reach out to them and ask for eligibility to provide a free / "open source" tier.

Choosing which feature to implement next, finding collaborators / contributors, dealing with external noise etc were some of the personal tough decisions I took while building Hoppscotch.

https://github.com/hoppscotch/hoppscotch
